proxies/shared/resources/jsc/NhsAppAccounts.Ok.Response.js

```diff
@@ -0,0 +1,41 @@
+const content = context.getVariable("response.content")
+
+const virtualhosts = context.getVariable("virtualhost.aliases");
+const hostname = virtualhosts.slice(-1)[0];
+const internalHostname = context.getVariable("request.header.host");
+
+const reqUrl = context.getVariable('proxy.url');
+const baseUrl = reqUrl.split("?")[0].replace(internalHostname, hostname);
+
+const parsedContent = JSON.parse(content)
+var odsOrganisationCode = null;
+
+const replacePathRoot = (link) => {
```

Comment:
Could this function be used in `Messages.Create.Response.js`

Reply:
Done